### DISCUSSION TITLE
Layout div classes do nothing, present in current stackblitz demo too

### DISCUSSION THREAD

**Discussion by vinzenzzinecker98** [primefaces/community#3005] [PrimeNG] [answered]
I thought it was my project setup, I double checked everything and nothing worked. Now I find that the same issue is present in the official Stackblitz demo:

- Layouts from div classes, that should be inherited from css are doing nothgin
- for example <div class="grid grid-nogutter"> etc. etc.

Take for example the dataview component: 
https://primeng.org/dataview (click on the flash to open stackblitz demo)

How it should look:
<img width="802" alt="image" src="https://github.com/user-attachments/assets/416fee88-17ad-4df9-a12f-875a7a2dea4b">

How it looks: 

<img width="632" alt="image" src="https://github.com/user-attachments/assets/d2ab7800-d096-48ab-9ac7-3489336eec46">

Do you have any ideas what I can do to fix that? Given that it is present in the official Demo, it could be a version issue?

Thanks in advance!

**Comment by lipesanmartin** [comment, marked as answer]
Do you have Tailwind CSS installed on your project? Not sure if it's the solution, but I'm using this component with Tailwind CSS installed on my project and it works perfectly. I think this component uses Tailwind classes for styling. It's pretty easy to use if you're not familiar with. I'd give it a try

**Comment by vinzenzzinecker98** [comment]
Locally it was wrong config of tailwind, but it seems that should be included in the stackblitz demos, also? 

Thanks anyway!

**Reply by lipesanmartin** [reply]
I find these StackBlitz demos very inconsistent. Some components don't even build; they break and throw errors beforehand. I think they should really improve this platform for testing.
Glad I could help!
